### ISSUE TITLE
Use rever to release?

### ISSUE THREAD

**Issue by CJ-Wright** [drdoctr/doctr#283]
I'm more than happy to PR, or start a PR. I just wanted to use this to gauge interest.

**Comment by asmeurer** [comment]
Yeah, I'd like to do it, if it can handle it. This is the release process. https://drdoctr.github.io/releasing.html 

Everything except for updating the changelog could be automated, although personally the most annoying step is the conda-forge step. 

One thing I think hasn't been implemented in rever yet is checking if Travis has passed before doing the release (I mentioned this to Anthony).

**Comment by CJ-Wright** [comment]
Why can't the changelog changes be automated?

**Comment by asmeurer** [comment]
The change log has to be handwritten one way or another. I'm happy with the current process (writing it at release time). If the development increases substantially I might prefer a different process, but for now, doing it all at once is preferred to doing it per PR. 

I guess the process of listing every merged PR since the previous release could be automated.

**Comment by CJ-Wright** [comment]
Hmm ok.
